client/src/context.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const Context = React.createContext();
+
+class AuthProvider extends React.Component {
+  state = {
+    isAuth: false,
+    logged: false,
+    admin: false,
+  };
+
```

Comment:
You need to go fetch the user current state: 
 ```
{
   logged: false,
   admin: false
  }
```

and there isnt currently a route, so you have to make one that sends the above data, something like:
`res.send({...req.user})`